netty/src/test/java/io/grpc/netty/NettyChannelBuilderTest.java

```diff
@@ -92,14 +92,35 @@ private void overrideAuthorityIsReadableHelper(NettyChannelBuilder builder,
   }
 
   @Test
+  @SuppressWarnings("deprecation")
```

Comment:
Instead, you can mark this method as `@Deprecated`. That can be a bit more clear that we planned on deleting this method (even if it is a test). Not a big deal in a test though.